src/JsonSchema/Iterator/ObjectIterator.php

```diff
@@ -49,7 +50,7 @@ public function current()
     /**
      * {@inheritdoc}
      */
-    public function next()
+    public function next(): void
```

Comment:
In it's current state the library supports `>=5.3.3` as per`composer.json` requirements. The `void` keyword was introduced only in PHP 7.1. I know there are plans to drop support for PHP <7.2 (Even approved by @Seldaek) but in an attempt to revive the library and doing some initial triage I did want to point this out for ourselves.

It also applies to the `int` and `bool` return types which where added in PHP 7.0.

In order to make this PR considered for merging into the repo changing all to `#[\ReturnTypeWillChange]` would be needed I guess.

Reply:
@R4c00n in an attempt to cleanup this repo we are trying to filter the pull request and see which ones might be closed and which ones we can help resolve. Do you still feel like pursuing this PR or did time catch up on us and are you working on other priorities? Feel free to close it yourself or comments if helpful.